cypress/e2e/court-cases/CourtCaseDetails.cy.ts

```diff
@@ -310,91 +280,54 @@ describe("Court case details", () => {
-    ])
-
-    const triggers: TestTrigger[] = [
-      {
-        triggerId: 0,
-        triggerCode: "TRPR0001",
-        status: "Unresolved",
-        createdAt: new Date("2022-07-09T10:22:34.000Z")
-      }
-    ]
-    cy.task("insertTriggers", { caseId: 0, triggers })
-
-    cy.login("bichard02@example.com", "password")
-    cy.visit("/bichard/court-cases/0")
-    cy.findByText(`Trigger locked by: ${userWithGeneralHandlerPermission}`).should("exist")
-    cy.findByText(`Error locked by: ${userWithGeneralHandlerPermission}`).should("exist")
-
-    cy.get("button").contains("Resolve trigger").click()
-    cy.get("table")
-      .eq(-1)
-      .then((table) => {
-        expect(table.find("td").eq(2).text()).to.equal(userWithGeneralHandlerPermission)
-        const resolutionTime = parse(table.find("td").eq(3).text(), "dd/MM/yyyy HH:mm:ss", new Date())
-        const minsSinceResolved = differenceInMinutes(new Date(), resolutionTime)
-        expect(minsSinceResolved).to.be.at.least(0)
-        expect(minsSinceResolved).to.be.below(1)
-      })
-  })
-
-  it("should resubmit a case when the resubmit button is clicked", () => {
-    cy.task("insertCourtCasesWithFields", [
-      {
-        errorLockedByUsername: null,
-        triggerLockedByUsername: null,
-        orgForPoliceFilter: "02"
-      }
-    ])
-
-    cy.login("bichard02@example.com", "password")
-
-    cy.visit("/bichard/court-cases/0")
-
-    cy.get("button").contains("Resubmit").click()
-    cy.location().should((loc) => {
-      expect(loc.href).to.contain("?courtCaseId=0&resubmitCase=true")
-    })
-
-    cy.get("H1").should("have.text", "Case details")
-    cy.contains("Notes").click()
-    const dateTimeRegex = /\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2}/
-    cy.contains(dateTimeRegex)
-    cy.contains("Bichard02: Portal Action: Resubmitted Message.")
-  })
-
-  it("should resubmit a case when updates are made and the resubmit button is clicked", () => {
-    cy.task("insertCourtCasesWithFields", [{ orgForPoliceFilter: "02" }])
-    cy.login("bichard02@example.com", "password")
-
-    cy.visit("/bichard/court-cases/0")
-
-    // need to make the updates and then check them in the db
-    cy.get("input").first().type("2024-09-26")
-    cy.get("button").contains("Resubmit").click()
-    cy.task("getCourtCaseById", { caseId: 0 }).then((res) =>
-      expect(JSON.stringify(res)).to.eq(JSON.stringify(resubmitCaseJson))
-    )
-    cy.location().should((loc) => {
-      expect(loc.href).to.contain("?courtCaseId=0&resubmitCase=true")
-    })
-
-    cy.get("H1").should("have.text", "Case details")
-    cy.contains("Notes").click()
-    const dateTimeRegex = /\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2}/
-    cy.contains(dateTimeRegex)
-    cy.contains("Bichard02: Portal Action: Update Applied. Element: nextHearingDate. New Value: 2024-09-26")
-    cy.contains("Bichard02: Portal Action: Resubmitted Message.")
-  })
+  // it("should resubmit a case when the resubmit button is clicked", () => {
```

Comment:
We need to uncomment these tests once the new tabs in the case details page are implemented.